tools/subgraph-example/hardhat.config.ts

```diff
@@ -62,6 +65,88 @@ task("mintERC721", "Mints an ERC721 token to a recipient", async (taskArgs, hre)
   await mintNFT(<string>process.env.RECEIVER_PRIVATE_KEY, hre);
 });
 
+task("deployHTS", "Deploys HTS Fungible Token", async (taskArgs, hre) => {
+  const wallet = new Wallet(
+    process.env.OPERATOR_ID!,
+    process.env.OPERATOR_KEY!,
+    new LocalProvider({client: Client.forNetwork(JSON.parse(process.env.HEDERA_NETWORK!))})
+  );
+  const contractName = "ExampleHTS";
+  await hre.run("compile");
+  let createTransaction = await new TokenCreateTransaction()
+    .setTokenName("ffff")
+    .setTokenSymbol("F")
+    .setDecimals(3)
+    .setInitialSupply(0)
+    .setTreasuryAccountId(wallet.getAccountId())
+    .setAdminKey(wallet.getAccountKey())
+    .setFreezeKey(wallet.getAccountKey())
+    .setWipeKey(wallet.getAccountKey())
+    .setSupplyKey(wallet.getAccountKey())
+    .setFreezeDefault(false)
+    .freezeWithSigner(wallet);
+
+  createTransaction = await createTransaction.signWithSigner(wallet);
+  const resp = await createTransaction.executeWithSigner(wallet);
+
+  const receipt = (await resp.getReceiptWithSigner(wallet));
+  const address = `0x${receipt.tokenId?.toSolidityAddress()}`;
```

Comment:
Q: Why are we using the SDK here?
Should this all be possible via ethers?
No need to change now but something we should revisit if there's no limitation

Reply:
Ethers can make only EthereumTransaction, here we need HAPI, so we need to use SDK.

Reply:
i see, in order for them to be synthetic we need to make them using HAPI 👍